elections/2024-SC/election.yaml

```diff
@@ -0,0 +1,17 @@
+name: Knative 2024 SC
+organization: Knative
+start_datetime: 2024-12-05 00:00:01
+end_datetime: 2024-12-20 23:59:00
+no_winners: 2
+allow_no_opinion: True
+delete_after: True
+show_candidate_fields:
+  - affiliation
+  - owners
+election_officers:
+  - geekygirldawn
+  - jberkus
+  - creydr
+eligibility: All Knative contributors with more than 25 contributions in the last 12 months are eligible to vote
```

Comment:
I see two sources for this with different eligibilities:

> Anyone who has at least 25 contributions in the last 12 months is eligible to vote in the SC election

from https://github.com/knative/community/blob/main/mechanics/SC.md#voter-eligibility

vs 

> Anyone who has at least 50 contributions in the last 12 months is eligible to vote in the Steering election

from https://github.com/knative/community/blob/main/STEERING-COMMITTEE.md#eligibility-to-vote

Which one is correct? @knative/steering-committee

Reply:
Good question!  Looking at the history, https://github.com/knative/community/pull/1452 and https://github.com/knative/community/pull/1451 seem to have been aimed to adjust the threshold to 25 contributions, so we should correct this in the `STEERING-COMMITTEE.md` file.

Reply:
:+1: Addressed in https://github.com/knative/community/pull/1632